### ISSUE TITLE
[FEATURE] form inputs components

### ISSUE THREAD

**Issue by OmarAbouzeid95** [khazifire/DripUI#62]
I was thinking of adding a couple of input fields for forms, what do you think?

**Comment by mayurjadhav2002** [comment]
Yes there should be more components for form inputs, selects, dropdowns and checkboxes. I'd also love to add more if this repo is still being maintained.
